src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/TransformationControllerTest.java

```diff
@@ -13,33 +13,42 @@
 import static java.util.Collections.emptyList;
 import static org.hamcrest.Matchers.allOf;
 import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.notNullValue;
 import static org.springframework.http.MediaType.APPLICATION_JSON;
-import static org.springframework.test.web.servlet.ResultMatcher.matchAll;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @WebMvcTest(TransformationController.class)
 public class TransformationControllerTest extends BaseControllerTest {
+
+    private static final String EXCEPTION_RECORD_JSON_PATH = "/fixtures/model/exception-record.json";
     private static final String API_URL = "/transform-exception-record";
     ObjectMapper objectMapper = new ObjectMapper();
 
     @Test
     public void shouldReturnSuccessResponseForTransformationEndpoint() throws Exception {
         JsonNode formToValidate = objectMapper.readTree(new File(getClass()
```

Comment:
Divorce review

Reply:
This seems to be the way it's done everywhere in FR, the `ResourceLoader` we know and love form divorce is not used anywhere